dev/com.ibm.ws.kernel.boot.common/src/com/ibm/ws/kernel/boot/classloader/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 1.0.23
+ */
+@org.osgi.annotation.versioning.Version("1.0.23")
```

Comment:
For packages, I think these usually start at "1.0.0" since the packages can evolve independently of the bundle version.

Reply:
I made it 1.0.0.